src/EnergyPlus/WeatherManager.hh

```diff
@@ -107,6 +107,9 @@ namespace WeatherManager {
     extern int const WP_ScheduleValue;  // User entered Schedule value for Weather Property
     extern int const WP_DryBulbDelta;   // User entered DryBulb difference Schedule value for Weather Property
     extern int const WP_DewPointDelta;  // User entered Dewpoint difference Schedule value for Weather Property
+    extern int const WP_BruntModel;  // Use Brunt model for sky emissivity calculation
```

Comment:
Do these all need to be `extern`? Looks like they're only used in WeatherManager.

Reply:
@mjwitte I "extern" them for the unit test only (otherwise they cannot be declared in the unit test file). Any better suggestions?

Reply:
The ideal option would be to just use an enum class for options like this.  There are occasions where that is not feasible, such as if a variable of that enum type is used as an integer output variable.  But those are very few, and in most cases, changing to an enum can fix this.  I'm not pushing for that now, but in the future, that would be great.